### DISCUSSION TITLE
Replace start button on W11 taskbar?

### DISCUSSION THREAD

**Discussion by Caspid** [Open-Shell/Open-Shell-Menu#1818] [Q&A]
I'm trying to replace the start button on the W11 taskbar, and the new button seems to appear, but I can't figure out how to hide the default Windows button.

![Screenshot 2024-01-10 065741](https://github.com/Open-Shell/Open-Shell-Menu/assets/2336773/8cab267d-b843-4a94-b1d7-77a15e4f9ff4)

**Comment by ge0rdi** [comment]
We cannot hide default start button on Windows 11 at the moment.
It uses different framework (XAML) and so far we haven't figured out how to change elements in that framework.

The issue is tracked here https://github.com/Open-Shell/Open-Shell-Menu/issues/1631.

**Reply by MissCurlyBlonde** [reply]
![Untitled-1](https://github.com/Open-Shell/Open-Shell-Menu/assets/109482878/f4745437-e412-4028-90e7-2900d45e9abe)
What are the resolutions that we need to design the start button & taskbar if we want to skin them with a new texture, on Win11 @ 1920x1080 monitor resolution?

**Reply by ge0rdi** [reply]
I'd say it doesn't matter much. `Open-Shell` will scale the images to actual button/taskbar size.
